src/sentry/integrations/slack/webhooks/event.py

```diff
@@ -39,17 +47,28 @@ class SlackEventEndpoint(SlackDMEndpoint):
     XXX(dcramer): a lot of this is copied from sentry-plugins right now, and will need refactoring
     """
 
+    _METRIC_FAILURE_KEY = SLACK_EVENT_ENDPOINT_FAILURE_DATADOG_METRIC
+    _METRICS_SUCCESS_KEY = SLACK_EVENT_ENDPOINT_SUCCESS_DATADOG_METRIC
+
     authentication_classes = ()
     permission_classes = ()
     slack_request_class = SlackEventRequest
 
     def reply(self, slack_request: SlackDMRequest, message: str) -> Response:
         payload = {"channel": slack_request.channel_id, "text": message}
-        client = SlackClient(integration_id=slack_request.integration.id)
-        try:
-            client.post("/chat.postMessage", data=payload, json=True)
-        except ApiError as e:
-            logger.error("slack.event.on-message-error", extra={"error": str(e)})
+        if options.get("slack.event-endpoint-sdk"):
```

Comment:
I am using an option here which isn't scoped to an org b/c adding an extra 1/2 RPC call will take longer and i am afraid we will start timing out.

Reply:
how do you plan to LA this?